packages/datadog-plugin-cypress/src/support.js

```diff
@@ -9,8 +9,6 @@ beforeEach(() => {
 afterEach(() => {
   const currentTest = Cypress.mocha.getRunner().suite.ctx.currentTest
   cy.task('dd:afterEach', {
-    testName: currentTest.fullTitle(),
-    testSuite: Cypress.mocha.getRootSuite().file,
```

Comment:
why remove these?

Reply:
They don't seem to do anything other than duplicate data from `beforeEach`. I don't think test can change title or suite during run.
Am I missing something?

Reply:
yeah you're right. I believe these were added in case tests could run in parallel, but they're not used at the moment, so we can safely remove them 👍